pyproject.toml

```diff
@@ -38,6 +39,18 @@ dev = [
     "typing-extensions",
     "pre-commit"
 ]
+dev313 = [
```

Comment:
I think, you can use [inlude group mechanism](https://peps.python.org/pep-0735/#dependency-group-include) here

Reply:
found another way) with `uv sync` --no-install-package flag